### ISSUE TITLE
[vue/html-indent] wrong indentation reported with just one prop/attr 

### ISSUE THREAD

**Issue by sqal** [vuejs/eslint-plugin-vue#375]
**Tell us about your environment**

* **ESLint Version:** 4.16.0
* **eslint-plugin-vue Version:** 4.2.2

**Please show your full configuration:**

```js
module.exports = {
  root: true,
  extends: [
    'plugin:vue/recommended'
  ],
  parserOptions: {
    parser: 'babel-eslint'
  },
  plugins: ['vue'],
  rules: {
    'vue/html-indent': ['error', 2,  {
      alignAttributesVertically: true
    }],
    'vue/html-self-closing': ['error', {
      html: {
        void: 'never',
        normal: 'never',
        component: 'never'
      }
    }],
    'vue/max-attributes-per-line': ['error', {
      singleline: 3,
      multiline: {
        max: 1,
        allowFirstLine: false,
      }
    }],
  },
};
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
  <div>
    <my-component
      :foo="true">
    </my-component>
  </div>
</template>
```

[Reproduction online](https://mysticatea.github.io/vue-eslint-demo/#eJyNVstu3DgQ/JWBLrkMYWCPA2+AIMD+wOboCyW1JMYUqTTJ8SiG/32Loh6U5AB7sDGsbvazuqn3orI1Fbfi2VM/aOnp64u5XJ5rdZ9+4Gc/isr2gzVkfIIul1tj7d8vhedAL8Wi+JRrJjNPyc7zU2b92VWshqhAj8Gyv9TUyKD95T1eMbKn2+XLt2H4cn0xH/HurF9cCw6aXHF7R9DGwXnlLQsXBuLi9te1MFZU0pGoqdKSpVfQWgVaOifwp1qzYghWMglDrfhNbDfc1CdV46ym/Og/UfHS+Om6is4ziWerBVNLjwWsqQxtu0VekyZP4i43BIkJya3bASmRnvqSeC95pTEHtKpQ8KkiC4omePSoQ3EqT5xs7YWD9JCYHbgeHoeU6eFZitKiMtJET34vcdSrBWmk1r5jG9puhYKpDhZbbUupD6AyBsF+1lVl7lKrOlV2WFHGOUBZvHXKkxtktZagVw+qxQSBDuizl+VqztCbcGNfWr0gtvyJvLReVWzl5SplSkHRBiCO2fqCOdLNISEosIu9Zbm1zHfKiZIaC0buOB0MJmQ7IFFC92rRY2iUVoY2GZNEd0udQU42JBplkMR4QMF7mfM0mOBgV8uStoRnELzcIKREICG5Sg7JFeqEoITrMNAdyjqBTL+CQjajIp0Q3BTKGZlcpt75cSCb8rsHegL/WJUBzO3GoSOzRRilGNke20XUsFt5dadV0vleg9DWKdOKEgR/JR8buhboj0qxXTjvlWiixjx7K6rQCuP32K9gPW16P90jFshtFYtoLzE7S2ZOoLtiF1gfsDuqDtUxGL/B6intU2hxO4qBLe7HQZZuJ7Sg2YiRUmZabSGSZFZWWYRpIzVhKsJdgHD4r5pcvt8nGzjvlC2TXD7xMSf/jEeyR1fEbDkXMDniO4I8enLYUIDXx0TU0sudgqpJUNOAA+5/pLu8PtHRHn94eJJiKX8uPBI/wpZrdG52OIW2yRa2b1Grs3B+66Y4T8IITuNwvsfgIzwz+cDmJE1NzJNbJdOI/cFrtPVZ9TYz6ek90n7aVLi3lHUV3EUJVeH8qHMwEnkPpcFfusLW+oMsWTqBGF35ekKxrSgncI6fwCbj4ILFOT6Bnxjs8a10VszW0wZVZ88DnzEszLcTGGkJ8ONapML/q36nm+n4Y4xLt0jP2rUgfGxYnsH4NffN1P/EV+57/LK7FtNbg5yLMu51LG0sHl98/Acbs2wV)

**What did you expect to happen?**

I expect vue/html-indent error to not being reported in that case

**What actually happened? Please include the actual, raw output from ESLint.**

```
 4:1 Expected indentation of 4 spaces but found 6 spaces. (vue/html-indent)
```

But if you add another prop after foo then it works as expected:

```html
<my-component
  :foo="true"
  :bar="false">
</my-component>
```

Am i missing something in my configuration? 🤔

**Comment by mysticatea** [comment]
Thank you for the report.

I confirmed it, and I will fix it.

**Comment by iujlaki** [comment]
Hi,

We still have the problem with the same config above :(

**Comment by michalsnik** [comment]
Hey @iujlaki Are you sure you have the right version of eslint-plugin-vue installed? This [reproduction](https://mysticatea.github.io/vue-eslint-demo/#eJyNVstu3DgQ/JWBLrkMYWCPA2+AIMD+wOboCyW1JMYUqTTJ8SiG/32Loh6U5AB7sDGsbvazuqn3orI1Fbfi2VM/aOnp64u5XJ5rdZ9+4Gc/isr2gzVkfIIul1tj7d8vhedAL8Wi+JRrJjNPyc7zU2b92VWshqhAj8Gyv9TUyKD95T1eMbKn2+XLt2H4cn0xH/HurF9cCw6aXHF7R9DGwXnlLQsXBuLi9te1MFZU0pGoqdKSpVfQWgVaOifwp1qzYghWMglDrfhNbDfc1CdV46ym/Og/UfHS+Om6is4ziWerBVNLjwWsqQxtu0VekyZP4i43BIkJya3bASmRnvqSeC95pTEHtKpQ8KkiC4omePSoQ3EqT5xs7YWD9JCYHbgeHoeU6eFZitKiMtJET34vcdSrBWmk1r5jG9puhYKpDhZbbUupD6AyBsF+1lVl7lKrOlV2WFHGOUBZvHXKkxtktZagVw+qxQSBDuizl+VqztCbcGNfWr0gtvyJvLReVWzl5SplSkHRBiCO2fqCOdLNISEosIu9Zbm1zHfKiZIaC0buOB0MJmQ7IFFC92rRY2iUVoY2GZNEd0udQU42JBplkMR4QMF7mfM0mOBgV8uStoRnELzcIKREICG5Sg7JFeqEoITrMNAdyjqBTL+CQjajIp0Q3BTKGZlcpt75cSCb8rsHegL/WJUBzO3GoSOzRRilGNke20XUsFt5dadV0vleg9DWKdOKEgR/JR8buhboj0qxXTjvlWiixjx7K6rQCuP32K9gPW16P90jFshtFYtoLzE7S2ZOoLtiF1gfsDuqDtUxGL/B6intU2hxO4qBLe7HQZZuJ7Sg2YiRUmZabSGSZFZWWYRpIzVhKsJdgHD4r5pcvt8nGzjvlC2TXD7xMSf/jEeyR1fEbDkXMDniO4I8enLYUIDXx0TU0sudgqpJUNOAA+5/pLu8PtHRHn94eJJiKX8uPBI/wpZrdG52OIW2yRa2b1Grs3B+66Y4T8IITuNwvsfgIzwz+cDmJE1NzJNbJdOI/cFrtPVZ9TYz6ek90n7aVLi3lHUV3EUJVeH8qHMwEnkPpcFfusLW+oMsWTqBGF35ekKxrSgncI6fwCbj4ILFOT6Bnxjs8a10VszW0wZVZ88DnzEszLcTGGkJ8ONapML/q36nm+n4Y4xLt0jP2rUgfGxYnsH4NffN1P/EV+57/LK7FtNbg5yLMu51LG0sHl98/Acbs2wV) seems to be working fine 🤔

**Comment by iujlaki** [comment]
Hey @michalsnik, yep it's the same. It's weird because it works on several Mac but in Linux it doesn't.
The only differences is with the node and npm versions:
- node v8.10.0
- npm v5.6.0

**Comment by michalsnik** [comment]
Perhaps you're running global eslint and have a plugin installed globally as well?
Does it happen if you execute:
```
./node_modules/.bin/eslint . --ext .js,.vue
```
?

**Comment by iujlaki** [comment]
We got the same error :(
What should I help to find out?

**Comment by michalsnik** [comment]
Show me your `.eslintrc` content @iujlaki and what versions of eslint-plugin-vue and vue-eslint-parser you have installed :)
